content/hacking-atom/sections/debugging.md

```diff
@@ -297,3 +297,28 @@ You can then include the startup profile in any Issue you report.
 If you are having issues installing a package using `apm install`, this could be because the package has dependencies on libraries that contain native code. This means you will need to have a C++ compiler and Python installed to be able to install it. You can run `apm install --check` to see if the Atom package manager can build native code on your machine.
 
 Check out the pre-requisites in the [build instructions](https://github.com/atom/atom/tree/master/docs/build-instructions) for your platform for more details.
+
+#### Check if your GPU is causing the problem
+
+If you encounter flickering or other rendering issues, you can disable your Graphics Processing Unit (GPU) with the `--disable-gpu` Chromium flag to see if the fault lies with your GPU:
+
+```command-line
```

Comment:
`command-line` isn't a [known language](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml), so this won't highlight anything. I tend to use `ShellSession` since that's what you are documenting here, however as you can see below this snippet is too small for any of this to make a difference 😆.

----

`command-line`:
```command-line
$ atom --disable-gpu
```

`ShellSession`:
```ShellSession
$ atom --disable-gpu
```

`sh` (with `$`):
```sh
$ atom --disable-gpu
```

`sh` (without `$`):
```sh
atom --disable-gpu
```